run_dir/static/css/status.css

```diff
@@ -40,13 +40,13 @@ multiple places and typically control typography, or other common elements.
 	overflow: hidden;
 }
 
-#project_quotas td {
+.table-disk-quotas td {
```

Comment:
This particular chunk isn't doing anything currently, so it shouldn't make any difference if it's removed (as long as it doesn't cause a syntax error obviously). If you do the change I suggested above then it will remove the spacing that is around the bar in each plot. You can see this difference when comparing stage to production:

**Stage:**
![image](https://cloud.githubusercontent.com/assets/465550/13110837/996b4c1a-d582-11e5-8287-e1213a3ac3e7.png)


**Production:**
![image](https://cloud.githubusercontent.com/assets/465550/13110826/8fc47858-d582-11e5-8799-6cba19d1ea99.png)